src/pages/Main.js

```diff
@@ -49,12 +49,7 @@ export default hot(module)(class Main extends Component {
 
         Utils.getAuthInstance().isSignedIn.listen(status => this.setState({ isSignedIn: status }))
 
-        window.gapi.signin2.render('signInButton', {
-          scope: [
-            'profile', 'email', 'openid',
-            'https://www.googleapis.com/auth/compute'
-          ].join(' ')
-        })
+        window.gapi.signin2.render('signInButton', { scope: 'profile email' })
```

Comment:
Sorry, we're sure everything works as expected without the openid scope? I'm slightly nervous, because I thought it was what gave us the auth token, but I could certainly be wrong.

Reply:
https://developers.google.com/identity/protocols/OpenIDConnect makes it sound as if `openid` is sort of the underpinning for everything else